app/models/grade_entry_form.rb

```diff
@@ -243,7 +243,7 @@ def get_csv_grades_report
   # grades_file is the CSV file to be parsed
   # grade_entry_form is the grade entry form that is being updated
   # invalid_lines will store all problematic lines from the CSV file
-  def self.parse_csv(grades_file, grade_entry_form, invalid_lines, encoding)
+  def self.parse_csv(grades_file, grade_entry_form, invalid_lines, encoding, overwrite)
```

Comment:
Assignment Branch Condition size for parse_csv is too high. [46.27/15]<br>Line is too long. [87/80]